circe-testutils/src/main/scalajs/sttp/apispec/test/ResourcePlatform.scala

```diff
@@ -1,10 +1,11 @@
-package sttp.apispec.openapi.circe
+package sttp.apispec.test
 
 import io.circe.{Json, Error}
 import io.circe.parser.decode
 
 trait ResourcePlatform {
-  def rscPath(path: String): String = "openapi-circe/src/test/resources" + path
+  def basedir: String
```

Comment:
I think I'd add some comments on what this should point to
Plus `rscPath` stands for `resourcesPath`? A bit more elaborate name could help as well :)